newIDE/app/src/ObjectEditor/Editors/SpriteEditor/CollisionMasksEditor/index.js

```diff
@@ -28,7 +28,6 @@ import Paper from '../../../../UI/Paper';
 import ScrollView from '../../../../UI/ScrollView';
```

Comment:
I guess this file (and PointsEditor) could even be moved to a ObjectEditor/Editors/AnimatableEditor folder

Reply:
I'm not sure, maybe "ObjectEditor/Editors/AnimationListEditor" as 3D models and Spine objects are also Animatable but don't use this UI component.